mmv1/third_party/terraform/tests/resource_privateca_certificate_authority_test.go

```diff
@@ -6,6 +6,38 @@ import (
 	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
 )
 
+func TestAccPrivatecaCertificateAuthority_privatecaCertificateAuthorityIsStaged(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"pool_name":           BootstrapSharedCaPoolInLocation(t, "us-central1"),
+		"pool_location":       "us-central1",
+		"deletion_protection": false,
+		"random_suffix":       randString(t, 10),
+	}
+
+	resourceName := "google_privateca_certificate_authority.default"
+	vcrTest(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccCheckPrivatecaCertificateAuthorityDestroyProducer(t),
+		Steps: []resource.TestStep{
+			{
+				Config: testAccPrivatecaCertificateAuthority_privatecaCertificateAuthorityIsStaged(context),
+				Check: resource.ComposeTestCheckFunc(
```

Comment:
Any reason to not also add an import step here? with `is_staged` in the `ImportStateVerifyIgnore` field

Reply:
Close absolute comment.